spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/ReactiveResilience4JCircuitBreakerFactory.java

```diff
@@ -66,10 +67,29 @@ public ReactiveResilience4JCircuitBreakerFactory(CircuitBreakerRegistry circuitB
 	@Override
 	public ReactiveCircuitBreaker create(String id) {
 		Assert.hasText(id, "A CircuitBreaker must have an id.");
-		Resilience4JConfigBuilder.Resilience4JCircuitBreakerConfiguration config = getConfigurations()
+		return this.create0(id, id);
+	}
+
+	@Override
+	public ReactiveCircuitBreaker create(String id, String groupName) {
+		Assert.hasText(id, "A CircuitBreaker must have an id.");
+		Assert.hasText(groupName, "A CircuitBreaker must have a group name.");
+		return this.create0(id, groupName);
+	}
+
+	private ReactiveResilience4JCircuitBreaker create0(String id, String groupName) {
```

Comment:
Why create this new method?  Why not put it in the `create` method above?

Reply:
`DRY!` extract method `create0` used by two methods in case of duplicated codes.

Reply:
yes, this is one step too far. Overloading create with the impl going in `create(String id, String groupName)` is simpler and easier to understand. Please remove `create0()`